docs: refactor aspect docs (#2455)
marigold-ui/marigold: docs/src/pages/components/aspect.mdx

### Ratio Square and maximum width

The example shows how the `ratio` and the `maxWidth` props with a image component inside is to use. The `ratio` has his default value `square`.

```tsx preview file=components/Aspect/ratio-square.demo.tsx

```

### Ratio Widescreen

> An Example too show the `ratio` prop with the type `widescreen`.

Reviewer: ```suggestion
An Example that shows the `ratio` prop set to `"widescreen"`.
```

```tsx preview file=components/Aspect/widescreen-aspect.demo.tsx

```

### Map

> This Example use a Iframe inside the Aspect Component. The `ratio` prop has the `landscape` type.

Reviewer: ```suggestion
This Example use a Iframe inside the Aspect Component. The `ratio` prop is set to `"landscape"`.
```
